cdrtools checks for gcc masquerading as cc by looking at the output of
`cc -v`. However, it looks for the wrong string for modern gcc.

diff -wpruN '--exclude=*.orig' a~/conf/cc-config.sh a/conf/cc-config.sh
--- a~/conf/cc-config.sh	1970-01-01 00:00:00
+++ a/conf/cc-config.sh	1970-01-01 00:00:00
@@ -131,7 +131,7 @@ if [ $CC_FOUND = TRUE ]; then
 	# Call $CC and try to find out whether it might be "gcc" or "clang".
 	#
 	CC_V=`eval "$CC -v > /dev/null" 2>&1`
-	GCC_V=`echo "$CC_V" | grep -i gcc-version `
+	GCC_V=`echo "$CC_V" | grep -i "gcc version" `
 	CLANG_V=`echo "$CC_V" | grep -i clang `
 
 	if [ ".$GCC_V" != . ]; then
